src/subscript/prtvol2csv/prtvol2csv.py

```diff
@@ -233,9 +241,14 @@ def reservoir_volumes_from_prt(prt_file: str) -> pd.DataFrame:
     table_found = (
         False  # State determining if current line is in our interesting table or not.
     )
+    fipname_found = fipname == "FIPNUM"  # found the corrent fipname, FIPNUM OK
+
     with Path(prt_file).open(encoding="utf8") as f_handle:
         for line in f_handle:
-            if start_matcher.search(line) is not None:
+            if f"{fipname}  REPORT REGION" in line:
```

Comment:
Is it possible there could be instances of just a single space, or more than two?

Reply:
```python
if fipname in line and "REPORT REGION" in line:
     ...
```

One possible other option -- maybe not the best.

Reply:
I looked for another line in the PRT file for the same check, (a few lines above in the PRT file, at start of the report, "  BALZON"  for FIP-keyword FIPZON).  Seems quite robust, according to the Eclipse reference manual (see FIP-keyword).   Not sure how things will work with output from OPM Flow...